vocabularies/depth-reference.ttl

```diff
@@ -24,17 +20,17 @@
         drd:seismic-reference-datum,
         drd:water-bottom,
         drd:well-head ;
-    skos:prefLabel "Depth Reference Datum"@en .
+    skos:prefLabel "Depth Reference"@en .
     
 drd:absolute a skos:Collection ;
-    skos:definition "A datum that describes an absolute plane of reference at a global or regional scale."@en ;
+    skos:definition "A reference that describes an absolute plane of depth at a global or regional scale."@en ;

```

Comment:
To be consistent with change definition should be to describe an absolute plane _or point_
More accurately "A fixed plane or point that describes an absolute reference for depth observations.